cirq-core/cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -405,7 +405,7 @@ def cross_entropy_benchmarking(
         probs_exp_k = []  # type: List[np.ndarray]
         for circ_k in circuits_k:
             res = simulator.simulate(circ_k, qubit_order=qubits)
-            state_probs = np.abs(np.asarray(res.final_state_vector)) ** 2  # type: ignore
+            state_probs = cirq.state_vector_to_probabilities(np.asarray(res.final_state_vector))  # type: ignore
```

Comment:
Does the mypy check pass if we remove this `# type: ignore`?

Reply:
Yes it does apparently. I have removed it in the new commit